### PR TITLE
Added additional endpoints for example user.

### PR DESCRIPTION
**Pull request by moniakodzi**
Please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "name": "ideal-synergy",
   "version": "1.0.0",
+  "type": "module",
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start":"node –experimental-modules ./index.js",
```
Comment (codingguydynamite) [inline package.json]: This will be not user - please remove this line

**Review by kamilkodzi — COMMENTED** [review]
Remove unecessary scripts

File: package.json (hunk 1 above)
Comment (moniakodzi) [reply package.json]: Fixed

**Comment by kamilkodzi** [comment]
Brillant !

**Pull request merged**